src/main/java/net/minestom/server/utils/UUIDUtils.java

```diff
@@ -11,7 +11,7 @@
  * An utilities class for {@link UUID}.
  */
 @ApiStatus.Internal
-public final class UniqueIdUtils {
+public final class UUIDUtils {
```

Comment:
This is still a breaking change from my opinion

Reply:
I think using a marked internal class accepts a contract that it may change.

Reply:
IMO this does not need to be announced as such since it is internal api

Reply:
Some PPL use it because they don't understand what Internal means

Reply:
True, but they will learn. I think this is fairly obvious given that the IDE will warn of usages of internal apis, not just looking at the class itself. This is also how we have operated in the past.